### PR TITLE
added -v level logging to ftl/node/main.py

### PR DESCRIPTION
**Pull request by aaron-prindle**

### CODE REVIEW

File: ftl/node/main.py [hunk 1]
```diff
@@ -46,9 +46,14 @@
     action='store',
     help='The path where the application data sits.')
 
+parser.add_argument("-v", "--verbose",
+    const=1, default=0, type=int, nargs="?",
+    help="increase verbosity: 0 = only warnings, 1 = info, 2 = debug. " +
```
Comment (dlorenc) [inline ftl/node/main.py]: I kind of prefer using the actual strings (debug, warning, info) etc. as flags. WDYT?

Comment (aaron-prindle) [reply ftl/node/main.py]: sgtm

**Comment by aaron-prindle** [comment]
updated logging to use syntax:
main.py --v (DEBUG, INFO, etc.)

File: ftl/node/main.py [hunk 2]
```diff
@@ -46,9 +46,14 @@
     action='store',
     help='The path where the application data sits.')
 
+parser.add_argument("-v", "--verbose",
```
Comment (dlorenc) [inline ftl/node/main.py]: nit: --verbosity as the long form?

Comment (dlorenc) [reply ftl/node/main.py]: And variable name. You can use the "choices" parameter here to get the help text and validation for free.

Comment (aaron-prindle) [reply ftl/node/main.py]: both done

**Comment by dlorenc** [comment]
Looks like a flake issue.

**Comment by aaron-prindle** [comment]
fixed

**Pull request merged**